data/db.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Where is json-serverless? I can't find a file called that.

Reply:
This file was some dummy mock data for a mock service in case it was needed in the template. We don't seem be using that so I think we can remove that

(package.json)
"json-serverless": "1.6.14",
"mock-server": "jsonsls run ./data/db.json",

(readme)
[json-serverless](https://github.com/pharindoko/json-serverless) has been added to the repository should we wish to mock external services during development and can be used in conjunction with the `test` environment.